app/events/event_differ.rb

```diff
@@ -0,0 +1,104 @@
+module Types
+  include Dry.Types()
+end
+
+module EventDiffer
+  # Used to indicate that a storage location exists in one source but not the other.
+  class LocationDiff < Dry::Struct
+    attribute :storage_location_id, Types::Integer
+    attribute :database, Types::Bool
```

Comment:
I'm wondering if there is a different or more explicit name for this. Like... in_transactional_model or in_stateful_model or something. Food for thought

Reply:
I'm definitely down with brainstorming better names... the most explicit thing I could think of is "this is the thing that lives in the database" and "this is the thing that gets built via an aggregate". I'm not sure about something like "stateful" (an aggregate is state, it's just derived state) or transactional (things can live in the database without relying on transactions).